.github/workflows/release-master.yaml

```diff
@@ -72,6 +72,38 @@ jobs:
           path: /tmp/digests/*
           if-no-files-found: error
           retention-days: 1
+  helm:
+    permissions:
+      contents: read
+      packages: write
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: '3.12.3'
+          token: ${{ secrets.GH_PAT }}
+          
+      - name: Set Version
+        id: set_version
+        run: echo "VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
+
+      - name: Update Chart.yaml
+        run: |
+          sed -i "s/version: .*/version: $VERSION/" deploy/charts/kmesh-helm/Chart.yaml
+
+      - name: Update Image Tag
+        run: |
+          sed -i "s/image: .*/image: my-image:$VERSION/" deploy/kubernetes/deployment.yaml
```

Comment:
This is not the right place, you should change the helm chart dir, and the tag in values.yaml